added a bit more detail to run and invoke; minor wording updates (#80)
capeprivacy/pycape: README.md

## Usage

Before running a function, you need to first get an access token with the [Cape CLI](https://github.com/capeprivacy/cli) by running `cape login`. Once logged into Cape, your token can be found in the `~/.config/cape/auth` file. The access token will be used when instantiating a `Cape` client. If the `access_token` attribute is None, it will try to automatically load the access token from your config file. You'll also need a function id (`function_id`) and (optionally) function hash (`function_hash`), which get returned when you deploy a function with the CLI by running `cape deploy`.

### `run`

> Run is used to invoke a function once with a single input. A connection to a Cape function is created, then terminated upon completion (no set up or tear down is required). If you wish to invoke the same function multiple times without terminating the connection between calls, please see [invoke](https://github.com/capeprivacy/pycape/#invoke). If the `function_hash` attribute is None, then verification of the function hash will not occur. It is encouraged to always provide the function hash for better security. By default, inputs and outputs are expected to be bytes.

Reviewer: ```suggestion
Run is used to invoke a function once with a single input. A connection to a Cape function is created, then terminated upon completion (no set up or tear down is required). If you wish to invoke the same function multiple times without terminating the connection between calls, please see [invoke](#invoke). If the `function_hash` attribute is None, then verification of the function hash will not occur. It is encouraged to always provide the function hash for better security. By default, inputs and outputs are expected to be bytes.
```

> > Note: You can optionally can use [Serdio](https://github.com/capeprivacy/pycape/tree/main/serdio) to help with serialization and deserialization of inputs and outputs. To learn more, please check out [this example](https://github.com/capeprivacy/pycape/tree/main/examples#mean-running-functions-on-python-types).

Reviewer: ```suggestion
> Note: You can optionally use [Serdio](https://github.com/capeprivacy/pycape/tree/main/serdio) to help with serialization and deserialization of inputs and outputs. To learn more, please check out [this example](https://pydocs.capeprivacy.com/walkthrough.html#mean-v2-running-functions-on-python-types-with-serdio).
```

### `invoke`

> Invoke is used to run a function repeatedly with multiple inputs. The connection to your Cape function is not terminated between invocations. It gives you more control over the lifecycle, and can be more efficient. Prior to calling `invoke`, `connect` to your function and then `close` it when you are finished.

Reviewer: ```suggestion
Invoke is used to run a function repeatedly with multiple inputs. The connection to your Cape function is not terminated between invocations. It gives you more control over the lifecycle, and can be more efficient. Prior to calling `invoke`, `connect` to your function and then `close` it when you are finished. You can also call `invoke` inside of a `Cape.function_context`, which will handle connecting and closing the connection for you. See the [docs](https://pydocs.capeprivacy.com/pycape.html#pycape.Cape.function_context) for a usage example.
```
